File: Application.API/Models/ConfigList.cs
using System.Collections.Generic;

namespace Application.API.Models {
    public class ConfigList {
        public ICollection<string> religion { get; set; }
        public ICollection<string> politics { get; set; }
        public ICollection<string> subChapter { get; set; }
        public ICollection<string> civilIdMouhavaza { get; set; }
        public ICollection<string> civilIdKadaa { get; set; }
        public ICollection<string> civilIdRegion { get; set; }
        public ICollection<string> civilIdPlace { get; set; }
    }
}

File: Application-SPA/src/app/_models/configList.ts
export interface ConfigList {
  religion: string[];
  politics: string[];
  subChapter: string[];
  civilIdMouhavaza: string[];
  civilIdKadaa: string[];
  civilIdRegion: string[];
  referenceUsers: SelectOnject[];
}
export interface SelectOnject {
  value: number;
  name: string;
}
